### ISSUE TITLE
This is issue already was reported as #38. In addition to that you do not need to yell in issue titles, this is not a YouTube video.

### ISSUE THREAD

**Issue by Bobsonn** [jellysquid3/Sodium#1412]
This is issue already was reported as #38. In addition to that you do not need to yell in issue titles, this is not a YouTube video.

_Originally posted by @LambdAurora in https://github.com/CaffeineMC/sodium-fabric/issues/1411#issuecomment-1203098205_

I understand that the problem has not been solved for 2 years?

**Comment by LambdAurora** [comment]
You do not need to open a new issue to continue discussion.

But to answer your question: it's a very difficult problem to solve **efficiently**.
The maintainers time are not infinitely extensible and all the mods you see here are most often made in people's free time.

Hopefully it'll be fixed on day, but for now it remains unfixed and all we can do is patiently wait for either someone to contribute an efficient fix, or a maintainer to come up with a fix.

tl;dr this is volunteer's work and it takes time, volunteers' time is slim.

**Comment by Bobsonn** [comment]
I'm new, I don't know what's going on, sorry

**Comment by Bobsonn** [comment]
Is there an approximate date when the bug will be fixed?

**Comment by LambdAurora** [comment]
> I'm new, I don't know what's going on, sorry

It's ok, next time just keep in mind to look around in the bug tracker first, especially the pinned issues.
Sometimes researching saves time to everyone ;)

> Is there an approximate date when the bug will be fixed?

We don't do ETAs because they are very difficult to approximate, in addition to being very stressful for anyone working on it.
And as said, it's all volunteers's time, ETAs would end up as counter-productive.

The best thing to do is to sit down, relax, and wait.
A fix will come out at one point, it's just that no one knows when.
